tools/nx-plugins/npm-package/build/set-package-defaults.ts

```diff
@@ -52,6 +51,9 @@ export const setPackageDefaults = async (
 			type: 'git',
 			url: 'git+https://github.com/guardian/csnx.git',
 		},
+		engines: {
+			node: '>=16',
+		},
```

Comment:
It’s marked as stable and supported [as early as v12](https://nodejs.org/docs/latest-v12.x/api/esm.html)

Reply:
was only added without a flag in https://nodejs.org/fr/blog/release/v12.17.0. sindrsorhus's [recommendation is to start with node 16](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#how-can-i-move-my-commonjs-project-to-esm) and iirc, there's some complications with node 14 too. 

given [16 is EOL in just over a month from now](https://nodejs.org/en/blog/announcements/nodejs16-eol), this seems ok to me?